test/controllers/InternationalAddressControllerSpec.scala

```diff
@@ -43,8 +43,12 @@ class InternationalAddressControllerSpec extends SpecBase with MockitoSugar {
     userAnswersId,
     Json.obj(
       InternationalAddressPage.toString -> Json.obj(
-        "line1" -> "value 1",
-        "line2" -> "value 2"
+        "line1" -> "Testing Lane",
+        "line2" -> "Testing Road",
+        "city" -> "New York",
+        "region" -> None,
```

Comment:
I'd suggest just leaving the region out of this altogether if you want it to be "empty"